test.js

```diff
@@ -497,6 +497,44 @@ test('Nevercode - Not PR', function (t) {
   t.end()
 })
 
+test('GitHub Actions - PR', function (t) {
+  process.env.GITHUB_ACTIONS = 'true'
+  process.env.GITHUB_EVENT_NAME = 'pull_request'
+
+  clearModule('./')
+  var ci = require('./')
+
+  t.equal(ci.isCI, true)
+  t.equal(ci.isPR, true)
+  t.equal(ci.name, 'GitHub Actions')
+  t.equal(ci.GITHUB_ACTIONS, true)
+  assertVendorConstants('GITHUB_ACTIONS', ci, t)
+
+  delete process.env.GITHUB_ACTIONS
```

Comment:
You might want to perform this global teardown in a `finally` block in case any of the previous steps throw.